odin/adapters/iac_dummy.py

```diff
@@ -0,0 +1,83 @@
+import logging
```

Comment:
To avoid proliferation of test code source files in the main source tree I suggest we, as a minimum, merge iac_dummy.py and iac_dummy_target.py. They could also be merged into dummy.py.